### PR TITLE
feat(chat-e2e): update user agents for e2e tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

update user agents for e2e tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/ui/pages/dialHomePage.ts [hunk 1]
```diff
@@ -32,9 +32,15 @@ export class DialHomePage extends BasePage {
...
-    await appContainer
-      .getChatLoader()
-      .waitForState({ state: 'hidden', timeout: loadingTimeout });
+    //workaround for the issue https://github.com/epam/ai-dial-chat/issues/1596
+    try {
+      await appContainer
+        .getChatLoader()
+        .waitForState({ state: 'hidden', timeout: loadingTimeout });
+    } catch (error) {
+      await this.reloadPage();
```
Comment (nartovm) [inline apps/chat-e2e/src/ui/pages/dialHomePage.ts:41]: We are going to reload it only once? Is that enough?

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/ui/pages/dialHomePage.ts (hunk 1 above)
Comment (irinakartun) [reply apps/chat-e2e/src/ui/pages/dialHomePage.ts:41]: the method is called recursively in the next line

**Review by nartovm — DISMISSED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

**Review by nartovm — APPROVED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2533.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**